### DISCUSSION TITLE
Cannot activate copilot from PyCharm

### DISCUSSION THREAD

**Discussion by vabburi** [community/community#63510] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have enterprise account with Cisco/Meraki and got permissions to use CoPilot. However, when I try to login to copilot from Pycharm, it says I do not have permissions. I verified that me enterprise email is listed in my Github profile and that I'm part of Cisco Organization in Github. Any help to resolve this error is highly appreciated !

**Comment by Mroczek1992** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> I have enterprise account with Cisco/Meraki and got permissions to use CoPilot. However, when I try to login to copilot from Pycharm, it says I do not have permissions. I verified that me enterprise email is listed in my Github profile and that I'm part of Cisco Organization in Github. Any help to resolve this error is highly appreciated !

Good
